lib/auth/trustedcluster.go

```diff
@@ -112,6 +120,76 @@ func (a *AuthServer) DeleteTrustedCluster(name string) error {
 	return nil
 }
 
+// addEnabledTrustedCluster does Trusted Cluster exchange and creates enabled
+// resources on the backend.
+func (a *AuthServer) addEnabledTrustedCluster(t services.TrustedCluster) error {
```

Comment:
why two functions, and not `addTrustedCluster()` with if inside?

Reply:
These functions seem to be duplicates of `EnableTrustedCluster` and `DisableTrustedCluster` what is their significance?